grails-app/views/locationType/create.gsp

```diff
@@ -41,7 +41,6 @@
                             </td>
                             <td valign="top" class="value ${hasErrors(bean: locationTypeInstance, field: 'description', 'errors')}">
                                 <g:select name="locationTypeCode" class="chzn-select-deselect" from="${org.pih.warehouse.core.LocationTypeCode.list()}"
-                                          noSelection="['':'']"
```

Comment:
What is the default value at this moment? Doesn't it throw any error?

Reply:
It defaults to the first item in the list. This is how we do it in party roles (which is where I got the idea from).

![image](https://github.com/user-attachments/assets/ad859937-2059-4b7f-a497-f086427a4a6b)

See how there's no X button for Party so you always have to select something. It can't be empty.